app/workers/articles/rss_reader_worker.rb

```diff
@@ -11,7 +11,7 @@ def perform
       return if SiteConfig.dev_to?
```

Comment:
Is the plan to push this out first, give it a test run, then remove this?

Reply:
Yes!